src/coreclr/vm/gccover.cpp

```diff
@@ -76,6 +76,8 @@ bool IsGcCoverageInterruptInstruction(PBYTE instrPtr)
 
 #if defined(TARGET_ARM64)
     instrVal = *reinterpret_cast<UINT32*>(instrPtr);
+#elif defined(TARGET_LOONGARCH64)
```

Comment:
can this be merged with `TARGET_ARM64` above?

Reply:
OK, Thanks